scripts/kind-with-registry.sh

```diff
@@ -89,6 +89,9 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
+  extraPortMappings:
+  - containerPort: 30950
```

Comment:
@michaelsauter Add a comment to indicate this port would be used for the webhook interceptor :)